resources/fileTemplates/internal/Magento Entity Edit Action Controller Class.php.ft

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
Looks like code doubling. https://github.com/magento/magento2-phpstorm-plugin/blob/mainline-entity-manager/resources/fileTemplates/internal/Magento%20Entity%20New%20Action%20Controller%20Class.php.ft looks almost the same. Why do we need 2 templates?

Reply:
@VitaliyBoyko Thanks, for your attention here. For now, we have little code duplication. But templates that might be changed for different reasons should be separated. It is not true duplication, as said Robert C. Martin. And I believe that it is true.